bare-metal-1.ks.in

```diff
@@ -0,0 +1,99 @@
+#version=DEVEL
+#test name: bare-metal-1
+
+# Use defaults for network, bootloader and storage.
+%ksappend network/default.ks
+%ksappend bootloader/default.ks
+%ksappend storage/default.ks
+
+# Install the prepared tar.
+liveimg --url=http://10.43.136.2/trees/rv/bmi/BP2-0.0.4.tar.xz
```

Comment:
I think this URL should be a fragment.

Reply:
I need the URL also in the `%post` script. Anyway, this test is be disabled by default because of the tar image. I have added a comment there to make it clear.